src/app/comparison-portal/layout.tsx

```diff
@@ -0,0 +1,16 @@
+import { Metadata } from 'next';
+
+import { Topbar } from '@/components/Topbar/Topbar';
+
+export const metadata: Metadata = {
+  title: 'Comparison Portal',
+};
```

Comment:
This should be improved. Align with @Esoteriker regarding [F-154](https://linear.app/world-food-programme-wfp/issue/F-157/improve-seo-and-metadata-on-each-page)

Reply:
I would say @Esoteriker can do it since the task is to do it for all pages. Also he hasn't started with the task yet

Reply:
I will check and handle this